litedict.py

```diff
@@ -53,6 +54,8 @@ def __init__(
 
         self.encoder = encoder
         self.decoder = decoder
+        self.writeback = writeback
+        self._writeback = {}
```

Comment:
Create only if writeback is True to avoid unused internal attributes.

```suggestion
        if self. writeback:
            self._writeback = {}
```

Reply:
Sure. It is small since it won't be used but I can make this change